src/link/MachO.zig

```diff
@@ -1267,7 +1267,7 @@ fn updateString(self: *MachO, old_str_off: u32, new_name: []const u8) !u32 {
 fn addPadding(self: *MachO, size: u64, file_offset: u64) !void {
     if (size == 0) return;
 
-    const buf = try self.base.allocator.alloc(u8, size);
+    const buf = try self.base.allocator.alloc(u8, @intCast(usize, size));
```

Comment:
`math.cast(usize, size)` is safer, but 4GB of padding are pretty much unheard of.

Reply:
Yeah i think in release mode we can just assume that the padding wont be bigger than 4GB and thus `@intCast` is fine?

Reply:
Then I'd change the parameter to use a `u32` instead.
The method is pretty inefficient though, it allocates some buffer on the heap only to write a (supposedly small?) number of zeros to a file.

Reply:
Yeah that would be better then indeed, well I think its actually a WIP by @kubkon

Reply:
I just kept it like this, because I think this will be changed anyway later by @kubkon.

Reply:
Yeah, this method will go away at some point but I didn't get to that point yet. Feel free to tweak it in the mean time though @FireFox317.